File: Api/src/FavoDeMel.Domain/Usuarios/Usuario.cs
using FavoDeMel.Domain.Common;

namespace FavoDeMel.Domain.Usuarios
{
    public class Usuario : Entity<int>
    {
        public string Nome { get; set; }
        public string Login { get; set; }
        public string Password { get; set; }
        public UsuarioPerfil Perfil { get; set; }
        public bool Ativo { get; set; }
    }
}

File: favodemel-web/src/app/models/usuario.ts
export class Usuario {
  id: any;
  nome: string;
  login: string;
  password: string;
  perfil: UsuarioPerfil;
  ativo: boolean;
  token?: string;
  comissao: any;
}

export enum UsuarioPerfil {
  Garcon = 1,
  Cozinheiro = 2,
  Administrador = 3,
}

export const DescricaoPerfil = {
  1: 'Garçom',
  2: 'Cozinheiro(a)',
  3: 'Administrador(a)'
};
